components/specification/samples/2010-04/filter.ome.xml

```diff
@@ -81,9 +81,6 @@
 					<EmissionFilterRef ID="Filter:6"/>
 				</LightPath>
 			</Channel>
-			<Channel ID="Channel:3">
-				<FilterSetRef ID="FilterSet:2"/>
-			</Channel>
```

Comment:
Primarily because it was not passing `xmlvalid`: there was a discrepancy between the defined `sizeC` and the number of `Channel` elements.